terraform/environments/mlra/main.tf

```diff
@@ -1,90 +1,38 @@
-module "lb-access-logs-enabled" {
-  source = "github.com/ministryofjustice/modernisation-platform-terraform-loadbalancer"
+module "alb" {
+  # source = "github.com/ministryofjustice/modernisation-platform-terraform-loadbalancer"
+  source = "./modules/ALB"
```

Comment:
Can this module name be lower-case please?

Reply:
done